accounts-cluster-bench/src/main.rs

```diff
@@ -335,8 +335,37 @@ fn run_rpc_bench_loop(
     let mut iters = 0;
     let mut last_error = Instant::now();
     let mut last_print = Instant::now();
+    fn flush_stats(
+        iters: &i32,
+        last_print: &mut Instant,
+        rpc_bench: &RpcBench,
+        stats: &mut RpcBenchStats,
+        thread: &usize,
+    ) {
+        info!(
+            "t({}) rpc({:?}) iters: {} success: {} errors: {}",
+            thread, rpc_bench, iters, stats.success, stats.errors
+        );
+        if stats.success > 0 {
+            info!(
+                " t({}) rpc({:?} average success_time: {} us",
+                thread,
+                rpc_bench,
+                stats.total_success_time_us / stats.success
+            );
+        }
+        if stats.errors > 0 {
+            info!(
+                " rpc average average errors time: {} us",
+                stats.total_errors_time_us / stats.errors
+            );
+        }
+        *last_print = Instant::now();
+        *stats = RpcBenchStats::default();
+    }
```

Comment:
Extracted this logic into a function so that I can call it in two places. I'm a Rust moron so please let me know if this isn't the idiomatic way. I tried creating a `|| {}` closure, but ran into trouble with `stats` since its mutated outside.